benchmarks/libxslt_xpath/build.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash -eu
+#
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# This would require an instrumented libgcrypt build.
+CRYPTO_CONF=--with-crypto
+CRYPTO_LIBS=-lgcrypt
+
+cd ../libxml2
+./autogen.sh \
+    --disable-shared \
+    --without-c14n \
+    --without-legacy \
+    --without-push \
+    --without-python \
+    --without-reader \
+    --without-regexps \
+    --without-sax1 \
+    --without-schemas \
+    --without-schematron \
+    --without-valid \
+    --without-writer \
+    --without-zlib \
+    --without-lzma
+make -j$(nproc) V=1
+
+cd ../libxslt
+./autogen.sh \
+    --with-libxml-src=../libxml2 \
+    --disable-shared \
+    --without-python \
+    $CRYPTO_CONF \
+    --without-debug \
+    --without-debugger \
+    --without-profiler
+make -j$(nproc) V=1
+
+for file in xpath fuzz; do
+    # Compile as C
+    $CC $CFLAGS \
+        -I. -I../libxml2/include \
+        -c tests/fuzz/$file.c \
+        -o tests/fuzz/$file.o
+done
+
+for fuzzer in xpath; do
+    # Link with $CXX
+    $CXX $CXXFLAGS \
+        tests/fuzz/$fuzzer.o tests/fuzz/fuzz.o \
+        -o $OUT/$fuzzer \
+        $LIB_FUZZING_ENGINE \
+        libexslt/.libs/libexslt.a libxslt/.libs/libxslt.a \
+        ../libxml2/.libs/libxml2.a \
+        $CRYPTO_LIBS
+
+    zip -j $OUT/${fuzzer}_seed_corpus.zip tests/fuzz/seed/$fuzzer/*
+done
+
+cp tests/fuzz/xpath.dict tests/fuzz/xpath.xml $OUT/
```

Comment:
Don't know if I want zips commited to this repo or if we should put them in GCB bucket. Need to think about this.

Reply:
I prefer zip here as it is more clear for people looking at benchmark directory. Going to GCS folder and gsutil might appear obvious to us, but not for the common crowd.

Reply:
I will switch it to GCS if this explodes things, right now we are using simple benchmarks with 10-20mb benchmarks.

Reply:
Yes but the annoying thing is that unless we clean up the history corpora will take up space forever.